stdlib/public/core/MemoryLayout.swift

```diff
@@ -50,6 +52,7 @@ public enum MemoryLayout<T> {
   /// When allocating memory for multiple instances of `T` using an unsafe
   /// pointer, use a multiple of the type's stride instead of its size.
   @_transparent
+  @_alwaysEmitIntoClient
```

Comment:
I assume that:
```swift
// TODO: Merge this back into the noncopyable variant once we have @_preInverseGenerics
@available(swift, obsoleted: 5.11) // Legacy ABI compatibility
@usableFromInline
internal static var size: Int {
  return Int(Builtin.sizeof(T.self))
}
```
means this AEIC will go away and we can mark it `@_preInverseGenerics`?

Reply:
That's my hope!